packages/chrome.extensions.vm/tools/chocolateyinstall.ps1

```diff
@@ -68,6 +68,9 @@ try {
     $tries += 1
   }
 
+  # Wait for an extra 60 seconds to make sure all extensions' tabs are opened already.
+  Start-Sleep -Seconds 60
```

Comment:
it seems to me like the time could be reduced a bit, but better to play it safe. :thinking: